taccsite_cms/templates/assets_site_delayed.html

```diff
@@ -15,6 +15,19 @@
 <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha384-xBuQ/xzmlsLoJpyjoggmTEz8OWUFM0/RC5BsqQBDX2v5cMvDHcMakNTNrHIW2I5f" crossorigin="anonymous"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+<script type="module">
+  import updateEmailLinkHrefs from '/static/site_cms/js/modules/updateEmailLinkHrefs.js';
+
+  const scopeElement = document.getElementById('cms-content');
+  const fakeText = '__REMOVE_THIS__';
+
+  updateEmailLinkHrefs(scopeElement, fakeText, {
+    user: 'data-user',
+    domain: 'data-domain',
+    subject: 'data-subject',
+    body: 'data-body'
+  });
```

Comment:
I pass in values that already match the defaults, because I think it can (1) show a developer what attributes are expected without them needing to open the file and/or (2) save time for an editor that needs to just change the data attribute (perhaps after copy-pasting this code to a more specific or external use-case).